Resources/Prototypes/_NF/Catalog/Fills/Items/weapon_cases_expedition.yml

```diff
@@ -80,11 +190,13 @@
       amount: 1
     - id: MagazinePistolSubMachineGun
       amount: 2
+    - id: MagazineBoxPistol
+      amount: 2
```

Comment:
That and also because as of right now non-nfsd players don't have a sure way of procuring pistol ammo (ammo fabs have a chance to spawn, so not guaranteed)

Reply:
Gotcha, thanks!